docs/source/membership/membership.md

```diff
@@ -172,9 +172,9 @@ A local MSP folder contains the following sub-folders:
 
   **Note:** If you are using a [Hardware Security Module(HSM)](../hsm.html) for key management, this folder is empty because the private key is generated by and stored in the HSM.
 
-* **signcert:** For a peer or orderer node (or in a client's local MSP) this folder contains the node's **signing key**. This key matches cryptographically the node's identity included in **Node Identity** folder and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.
+* **signcert:**  For a peer or orderer node (or in a client’s local MSP) this folder contains the node’s certificate issued by CA. The certificate represents the node's identity, and that that certificate's corresponding **private key** can be used to generate signatures which may be verified by anyone with a copy of that certificate.
```

Comment:
The signcert folder does not contain the private key. That's what the keystore folder is for.

Reply:
Yes, agreed. The line "The certificate represents the node's identity, and that that certificate's corresponding **private key** can be.." only speaks about corresponding private key of the certificate in signcert folder; it does not tell that private key resides inside signcert folder.